config/serverless.yml

```diff
@@ -86,6 +86,9 @@ xpack.spaces.allowFeatureVisibility: false
 # Only display console autocomplete suggestions for ES endpoints that are available in serverless
 console.autocompleteDefinitions.endpointsAvailability: serverless
 
+# Do not check the ES version when running on Serverless
+elasticsearch.ignoreVersionMismatch: true
```

Comment:
wouldn't this also blow up in production runs, because this can only be set if we run dev mode?

Reply:
I changed the schema validation to allow any value on serverless (no matter whether it's prod or dev): https://github.com/elastic/kibana/pull/168076/files#diff-1d5db3ef19f452febf37ec480c4e7bc0b05131f8e3b4e3497acce0a8477e6ff3R149